src/settings/appsettings.cpp

```diff
@@ -971,6 +971,22 @@ QByteArray AppSettings::defaultOcrLanguagesString()
     return {};
 }
 
+QByteArray AppSettings::ocrConfigPath() const
+{
+    return m_settings->value(QStringLiteral("OCR/ConfigPath"), defaultOcrConfigPath()).toByteArray();
+}
+
+void AppSettings::setOcrConfigPath(const QByteArray &string)
+{
+    m_settings->setValue(QStringLiteral("OCR/ConfigPath"), string);
+}
+
+QByteArray AppSettings::defaultOcrConfigPath()
+{
+    QString tesseractConfig = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) + QDir::separator() + QCoreApplication::applicationName() + QDir::separator() + "tesseract";
+    return tesseractConfig.toLocal8Bit();
```

Comment:
I think it should be empty by default since currently user need to create edit it manually.

```suggestion
    return {};
```